dapp/src/components/shared/Alert.tsx

```diff
@@ -22,7 +22,7 @@ import Spinner from "./Spinner"
 const STATUSES = {
   info: {
     icon: IconInfoCircle,
-    colorScheme: "gold",
+    colorScheme: "blue",
```

Comment:
This change affects all alerts in the dApp. We probably don't want that but it should be confirmed with Sorin. We should probably set `colorScheme` when we define `CONNECTION_ALERTS`. This solution seems the easiest until we have a styleguide.

<img width="300" alt="Screenshot 2024-12-10 at 09 07 11" src="https://github.com/user-attachments/assets/3977f538-0f62-4894-91d8-1463f2120cd2">

<img width="300" alt="Screenshot 2024-12-10 at 09 16 13" src="https://github.com/user-attachments/assets/858c5be0-354e-4b7f-b3a7-74eeb519633a">

Reply:
Fixed. Ref commit: 8f4faffa

<img width="538" alt="image" src="https://github.com/user-attachments/assets/41596ae2-0578-44b5-bc03-abff17fda427">
<img width="585" alt="image" src="https://github.com/user-attachments/assets/37defc19-1d4d-4b09-941d-e88427c3abbd">
<img width="581" alt="image" src="https://github.com/user-attachments/assets/14171ae0-7f37-4076-a799-68f5dcfde00f">